Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -152,22 +152,19 @@ private double getTagTrendingScore(Collection<Tag> tags) throws Exception {
     int numTags = tags.size();
     for (Tag tag : tags) {
       double tagScore = tag.getTrendingScoreTag();
-
-      // TODO: Use GoogleTrends API to update tag trending score
-
       sumScores += tagScore;
     }
     return (sumScores / numTags);
   }
 
+    //return Collection of all Tags in db
     public Collection<Tag> getTagsDb() throws Exception{
         Collection<Tag> tags= db.getAllTags();
         return tags;
     }
-
-    public void updateTagScores(Collection<Tag> tagsToUpdate) throws Exception {
-        for (Tag tag: tagsToUpdate) {
-            db.updateTag(tag);
-        }
+    
+    //updates one tag in db with a new trendingScore
+    public void updateTagScore(Tag tagToUpdate) throws Exception {
```

Comment:
why does this file need to update tag score. We should have a separate job which updates tag scores in db, this will only take care of reading it.

Reply:
The functions in this file are already connected to the db, and I thought making a new db client in a separate file would be additional overhead. Since reading and updating tags is required to find the trending charities, I thought that it would be okay to place these functions in this file. Please let me know if you want me to still move them into a separate file.

Reply:
we should ideally have such util functions part of util class.